Lib/codecs.py

```diff
@@ -847,6 +847,12 @@ def reset(self):
         self.reader.reset()
         self.writer.reset()
 
+    def seek(self, offset, whence=0):
+        # Seeks must be propogated to both the readers and writers
```

Comment:
Minor nitpick but there is a typo in the comment ("propogated" -> "propagated")

Reply:
Thanks, fixed.